app/config.gradle

```diff
@@ -1,6 +1,6 @@
 ext {
     compileSdkVersion = 25
-    buildToolsVersion = "26.0.2"
+    buildToolsVersion = "27.0.3"
```

Comment:
Not required in latest Android Gradle Plugin

Reply:
@batbrain7